src/author/author.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common';
+import { AuthService } from './author.service';
+import { CreateAuthhorDto } from './dto/create-author.dto';
+import { UpdateAuthorDto } from './dto/update-author.dto';
+import { AuthorInterface } from './interfaces/author.interface';
+import { FindOneAuthorInterface } from './interfaces/find-one-author.interface';
+
+@Controller('author')
+export class AuthorController {
+  constructor(private authService: AuthService) {}
+  @Post()
+  create(@Body() createAuthorDto: CreateAuthhorDto): AuthorInterface {
+    return this.authService.create(createAuthorDto);
+  }
+  @Get(':id')
+  findOne(@Param('id') id: number): FindOneAuthorInterface {
+    return this.authService.findOne(id);
+  }
+  @Get()
+  getAll(): AuthorInterface[] {
```

Comment:
Order Should Be create-findAll-findOne-update-remove in everywhere service repo and controller